src/App.js

```diff
@@ -1,16 +1,24 @@
 import React, { useState, useEffect } from 'react';
 import { getAllData } from './util/index';
 import UserRegistration from './components/UserRegistration';
+import SignIn from './components/SignIn';
 import Home from './components/Home';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import Logout from './components/Logout';
-import SignIn from './components/SignIn';
-import { Routes, Route } from 'react-router-dom';
-import { RequireAuth } from '@akosasante/react-auth-context';
+import Navbar from './components/Navbar';
+import {
+  AuthStatus,
+  RequireAuth,
+  useAuth,
+} from '@akosasante/react-auth-context';
 
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
   const [message, setMessage] = useState('');
+  const { status } = useAuth();
+  const isLoggedIn = status === AuthStatus.LoggedIn;
+  // const navigate = useNavigate();
```

Comment:
We can remove this if it's not being used (and remove it from the import as well)